nilearn/plotting/surf_plotting.py

```diff
@@ -29,10 +33,416 @@
 VALID_HEMISPHERES = "left", "right"
 
 
+AXIS_CONFIG = {
+    "showgrid": False,
+    "showline": False,
+    "ticks": "",
+    "title": "",
+    "showticklabels": False,
+    "zeroline": False,
+    "showspikes": False,
+    "spikesides": False,
+    "showbackground": False,
+}
+
+
+CAMERAS = {
+    "left": {
+        "eye": {"x": -1.7, "y": 0, "z": 0},
```

Comment:
`1.7` -> `1.5` allows for less padding and still fits for all views
```suggestion
        "eye": {"x": -1.5, "y": 0, "z": 0},
```